examples/simple-file-server/client/udp_file_client.bal

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/udp;
+import ballerina/lang.runtime as rt;
+
+// Port which the client is sending data.
+const int PORT = 20211;
+// Take the value of the terminal byte as 14.
+const int TERMINAL = 14;
+// Initialized the UDP client.
+udp:ConnectClient socketClient = check new ("localhost", PORT, timeout = 1);
```

Comment:
I think there is no point in making these module level variables.

Reply:
Moved into the `main` method.